CryptoTechReminderSystem/Gateway/HarvestGateway.cs

```diff
@@ -59,6 +59,21 @@ public IList<HarvestDeveloper> RetrieveDevelopers()
                 ).ToList();
         }
 
+        private bool CheckUserRole(JToken user)
+        {
+            var userRole = user["roles"].Select(role => role.ToString());
```

Comment:
😢

Reply:
Not sure what you mean here?

Reply:
Lots of userRole.contains here - not very scalable

Reply:
We've refactored the `IsDeveloper` method in this commit https://github.com/chubberlisk/crypto_tech/pull/30/commits/718368640db4514a307d2a941125490a15b42f7b. What do you think?

Reply:
@montashimo

Reply:
These changes look good. Have made a couple of other suggestions 👍